code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -188,6 +188,15 @@
 	build_path = /obj/item/storage/belt/medical
 	category = list("initial","Organic Materials")
 
+//SKYRAT EDIT ADDITION BEGIN
+/datum/design/medbandolier
+	name = "Medical Bandolier"
+	id = "medban"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 600)
+	build_path = /obj/item/storage/belt/medbandolier
+	category = list("initial","Organic Materials")
+//SKYRAT EDIT ADDITION END
```

Comment:
You don't even need this to be in this file, you can put it in a modular file in our modular_skyrat module, so this way, this change is modular.

Reply:
You forgot to remove this :^)